gluten-ut/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala

```diff
@@ -21,4 +21,8 @@ import org.apache.spark.sql.{GlutenTestConstants, GlutenTestsTrait}
 
 class GlutenMathExpressionsSuite extends MathExpressionsSuite with GlutenTestsTrait {
 
+  override def blackTestNameList: Seq[String] = Seq(
+    "bin"
+  )
+
```

Comment:
changes to
```
  override def blackTestNameList: Seq[String] = super.blackTestNameList ++ Seq(
    "bin"
  )
```
otherwise it will override parent method for CH Backend and run all test cases except 'bin' test case.

Reply:
Good catch!